### PR TITLE
Leap: Change div approach to use seq

### PR DESCRIPTION
**Pull request by keiravillekode**

### CODE REVIEW

**Comment by keiravillekode** [comment]
@clechasseur  - please feel free to add a "lgtm" review comment if all is good, or a "lgtm with nits" comment if you have specific change suggestions but it is otherwise good.

File: exercises/practice/leap/.approaches/using-rem/content.md [hunk 1]
```diff
@@ -27,7 +27,6 @@ set_leap:
         li      $v0, 1
 end:
         jr      $ra
-
```
Comment (keiravillekode) [inline exercises/practice/leap/.approaches/using-rem/content.md:30]: (One of the 4*2 source listings ended with an unneeded blank line.)

File: exercises/practice/leap/.approaches/using-macro/content.md [hunk 2]
```diff
@@ -1,4 +1,4 @@
-# Using the `andi` instruction
+# Using a "divisible by" macro
```
Comment (clechasseur) [inline exercises/practice/leap/.approaches/using-macro/content.md:1]: 🤦

**Review by clechasseur — APPROVED** [review]
Looks good! I was a little surprised that you asked me to review, since it was the first time I contributed anything to the track 😅 

Also, TIL the existence of the `seq` instruction 😉

**Review by ErikSchierboom — APPROVED** [review]

**Pull request merged**